.travis.yml

```diff
@@ -1,22 +1,26 @@
 language: go
 
+dist: trusty
+
 go:
-  - 1.12
+  - "1.12"
```

Comment:
Travis suggests forcing this to be a string, else you use golang 1.1 (via float interpolation to a string)